internal/pkg/earpointer/analysis.go

```diff
@@ -532,6 +597,41 @@ func (vis *visitor) visitBuiltin(builtin *ssa.Builtin, instr ssa.Instruction) {
 	}
 }
 
+// Handle some known functions, e.g. in package "fmt".
+func (vis *visitor) visitKnownFunction(fn *ssa.Function, instr ssa.Instruction) bool {
+	// Add an operand reference to a field of the "dst", i.e. dst[index -> op].
+	addField := func(dst ssa.Value, op ssa.Value, index int) {
+		fd := Field{Name: strconv.Itoa(index)}
+		if mayShareObject(op) {
+			for _, c := range vis.getContexts(dst) {
+				vis.unifyField(c, dst, fd, op)
+			}
+		}
+	}
+	fname := fn.Name()
+	if strings.HasPrefix(fname, "Sprint") || strings.HasPrefix(fname, "Errorf") {
```

Comment:
Consider using a `switch` with these prefix checks in the `case` clauses.

(Maybe you could put `return false` in the `default` case and `return true` after the `switch` to avoid duplication. Some may find this control flow unintuitive though, so maybe not.)